src/ro/onlinelibrary/people/readers/Adult.java

```diff
@@ -0,0 +1,51 @@
+package ro.onlinelibrary.people.readers;
+
+import ro.onlinelibrary.others.Address;
+import ro.onlinelibrary.people.BasicInfo;
+
+public class Adult extends BasicInfo {
+    private String employerCompany;
+    private String job;
+    private Address address;
+
+    public Adult(String firstName, String lastName, String employerCompany, String job, Address address) {
+        super(firstName, lastName);
+        this.employerCompany = employerCompany;
+        this.job = job;
+        this.address = address;
+    }
+
+    //accessors(getters) for first name, last name, employer company and job
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }

```

Comment:
why do you redeclare here the setters for firstName and lastName, you have them in the parent class, no need to redeclare them here